lighthouse-core/gather/gatherers/dobetterweb/external-anchors-with-no-rel-noopener.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('../gatherer');
+
+class ExternalAnchorsWithNoRelNoopener extends Gatherer {
```

Comment:
Yes. @paulirish's bit [here](https://github.com/GoogleChrome/lighthouse/pull/912#issuecomment-258655127) explains the reasoning. If users are opening their own site's links  (e.g. same origin) using `target="_blank"` then that's fine. They're in control of their own perf, but this audit is really for external links that kill the opening page's perf.